BraveShared/Preferences.swift

```diff
@@ -80,6 +80,16 @@ extension Preferences {
         public static let httpse = Option<String?>(key: "blockfile.httpse", default: nil)
     }
     
+    public final class ProductNotificationBenchmarks {
+        public static let trackerTierCount = Option<Int>(key: "product-benchmark.trackerTierCount", default: 0)
+        public static let allTiersShown = Option<Bool>(key: "product-benchmark.allTiersShown", default: false)
+        public static let firstTimeBlockingShown = Option<Bool>(key: "product-benchmark.firstTimeBlocking", default: false)
+        public static let privacyProtectionBlockShown = Option<Bool>(key: "product-benchmark.privacyProtectionBlockShown", default: false)
+        public static let httpsUpgradeShown = Option<Bool>(key: "product-benchmark.httpsUpgradeShown", default: false)
+        public static let videoAdBlockShown = Option<Bool>(key: "product-benchmark.videoAdBlockShown", default: false)
+        public static let ongoingEducationCheckDate = Option<TimeInterval>(key: "product-benchmark.ongoingEducationCheckDate", default: Date().timeIntervalSince1970)
```

Comment:
Store `Date` instead of `TimeInterval`?

Reply:
We can but I prefer TimeInterval here actually, easy to do calculations and  consistent with rest of the app.